priv/translations/af/LC_MESSAGES/relative_time.po

```diff
@@ -9,7 +9,6 @@
 msgid ""
 msgstr ""
 "Language: af\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
Ah, that makes more sense. I was perplexed by the messaging about the deprecation warning and assumed that the Plural-Forms header was no longer being supported at all, but on closer reading it's only generating the warning for certain ill-formed headers. In that case, I'll try to replace this PR with a more surgical one that fixes the format rather than removing plural-forms entirely, which I imagine was originally introduced by an GetText editing tool of some sort.

Reply:
@JasonTrue That is correct:

Either:

* Correct the plural forms header to a complete and valid one
* Or: remove it entirely and use the default known for the language

I recommend the first option since it helps with the tooling as well like Gettext file editors etc.

(I was part of making those changes in `gettext` / `expo`)